src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/BackupController.php

```diff
@@ -156,7 +133,9 @@ public function saveOptionsAction(Request $request)
             }
         }
 
-        return $this->redirectToRoute('admin_backups_index');
+        $backupForm = $this->getBackupFormHandler()->getForm();
```

Comment:
Really ? But the form is already in `$backupForm`.

Reply:
Yeah you are right, this is not needed here, fixed it.